bot/exts/backend/sync/_syncers.py

```diff
@@ -42,11 +40,7 @@ async def _sync(self, diff: _Diff) -> None:
         raise NotImplementedError  # pragma: no cover
 
     async def sync(self, guild: Guild, ctx: t.Optional[Context] = None) -> None:
-        """
-        Synchronise the database with the cache of `guild`.
-
-        If `ctx` is given, send a message with the results.
-        """
+        """If `ctx` is given, send a message with the results."""
         log.info(f"Starting {self.name} syncer.")
```

Comment:
Why did you change this? The original docstring was better.

Reply:
really confused about how that changed, will add the old one back.